presto-main/src/main/java/io/prestosql/dispatcher/QueuedStatementResource.java

```diff
@@ -468,11 +467,11 @@ public void destroy()
         }
 
         /**
-         * Instantiates an object that contains all information about query metadata.
+         * Builds the object that holds all metadata about query results.
```

Comment:
```suggestion
     * Instantiates the object that holds all metadata about query results.
```

I think it's better to use the word "Instantiate" instead of "Build", to avoid confusion with the Builder Design Pattern

Reply:
done